doc/usingchapel/privatebugs.rst

```diff
@@ -0,0 +1,36 @@
+.. _readme-privatebugs:
+
+===============================
+Reporting Private Chapel Issues
+===============================
+
+Private bug reports can be mailed directly to the Chapel team at Cray Inc.
+through :disguise:`chapel_bugs@cray.com`.
+
+
+When submitting a private bug report, please include the following information:
+
+.. note::
+
+    **Summary of Problem**
+
+    - What behavior did you observe when encountering this issue?
+    - What behavior did you expect to observe?
+
+    **Steps to Reproduce**
+
+    - Please attach source code that will reproduce the problem.
+        - To the extent possible, providing simplified programs demonstrating the
+          problem will be appreciated.
+        - If you are unable to submit code, please describe how to recreate the
+          issue.
+    - **How did you compile your program? e.g.** ``chpl foo.chpl -o foo``
```

Comment:
Bullets with sub-bullets get auto-bolded... I try to make this look more consistent by manually bolding this line.